### PR TITLE
Message for unsupported folder format [stage-2]

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@rodrigopavezi @stulees Please review. THx.

File: src/widget/storage-folder.js [hunk 1]
```diff
@@ -114,6 +114,15 @@ RiseVision.Image.StorageFolder = function (data) {
       RiseVision.Image.showError("The selected folder does not exist.");
     });
 
+
+    storage.addEventListener("rise-storage-folder-invalid", function () {
+      var params = { "event": "storage folder format(s) invalid" };
```
Comment (stulees) [inline src/widget/storage-folder.js]: @donnapep Image Widget should provide that as `error_details`. Need to change this to:

```
var params = { 
    "event": "error",
    "event_details: "storage folder format(s) invalid"
};
```

Comment (donnapep) [inline src/widget/storage-folder.js]: @stulees Guess that'll learn me to copy and paste. Fixed it. Please review again. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**